README.md

```diff
@@ -83,13 +86,16 @@ template. For more information, see the [example](./examples/splunk-wrapper/READ
    For more information about setting environment variables in the AWS console,
    see [Using AWS Lambda environment variables](https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html)
    on the AWS website.
-5. By default, the Splunk Lambda wrapper uses B3 context propagation. If you
-   want to change this, set the `OTEL_PROPAGATORS` environment variable in your
+5. By default, the Splunk Lambda wrapper uses B3 context propagation. 
+    
+    If you want to change this, set the `OTEL_PROPAGATORS` environment variable in your
    Lambda function code. For more information about available context
-   propagators, see the [Propagator settings](https://github.com/open-telemetry/opentelemetry-java-instrumentation#propagator)
-   for the OpenTelemetry Java Instrumentation.
-6. By default, the Splunk Lambda wrapper uses a jaeger-thrift exporter to send
-   traces to Splunk APM. If you want to use this exporter, set these environment
+   propagators, see the [Propagator settings](https://github.com/open-telemetry/opentelemetry-java/tree/v0.17.0/sdk-extensions/autoconfigure#customizing-the-opentelemetry-sdk)
+   for the OpenTelemetry Java.
+6. By default, the Splunk Lambda wrapper uses a `jaeger-thrift-splunk` exporter to send
+   traces to Splunk APM. 
+   
+   If you want to use this exporter, set these environment
    variables in your Lambda function code:
    ```
    OTEL_EXPORTER_JAEGER_ENDPOINT="http://yourEndpoint:9080/v1/trace"
```

Comment:
I can't add a comment on the next line, but the way `service.name` is now configured has also changed: it's now a part of the resource:
```sh
OTEL_RESOURCE_ATTRIBUTES="service.name=myServiceName"
```

Reply:
+1